lib/ReactViews/Custom/Chart/BottomDockChart.jsx

```diff
@@ -515,7 +515,7 @@ function calculateDomain(chartItems) {
   const ymax = Math.max(...chartItems.map(c => c.domain.y[1]));
   return {
     x: [xmin, xmax],
-    y: [Math.min(0, ymin), ymax]
+    y: [ymin, ymax]
```

Comment:
I have a suspicion that this is going to break some dataset, somewhere, that was depending on us clamping the min of the y domain to zero. But this is probably one of those situations where we tell people to fix their own data.

Reply:
Yeah, i think so too, but this shouldn't be the default behavior. If we find that we need it somewhere we can add a flag to force it per item. The problem with clamping automatically is that the when displaying a single chart that has y value >>> 0 the chart line looks flat loosing all the bumps and details which is bad.